src/secp256k1/contrib/lax_der_parsing.c

```diff
@@ -0,0 +1,150 @@
+/**********************************************************************
```

Comment:
I also don't know whether that will cause any issues with subtree-merging future upstream changes. I guess at worst it would cause a merge conflict if upstream alters the files.